apigw/src/shared/routes/auth/saml/index.ts

```diff
@@ -177,7 +183,13 @@ export default function createSamlRouter(config: SamlEndpointConfig): Router {
       req,
       'Logout callback called'
     )
-    await logoutExpress(req, res, config.sessionType)
+    const profile = await tryParseProfile(req, saml)
+    await logoutExpress(
+      req,
+      res,
+      config.sessionType,
+      profile?.nameID && createLogoutToken(profile.nameID, profile.sessionIndex)
```

Comment:
Replacing this with `undefined` should effectively simulate the situation in `master` when running the regression tests: if nothing is parsed from the SAML message, only cookies are used (and the tests don't currently care about `SameSite` in this context, so `evaka.eugw.session` will be available in the "IdP-initiated logout" test case).